sdk/src/types/block/payload/signed_transaction/transaction.rs

```diff
@@ -593,18 +593,13 @@ pub(crate) mod dto {
                 .network_id
                 .parse::<u64>()
                 .map_err(|_| Error::InvalidField("network_id"))?;
-            let mana_allotments = dto
-                .allotments
-                .into_iter()
-                .map(|o| ManaAllotment::try_from_dto_with_params_inner(o, params))
-                .collect::<Result<Vec<ManaAllotment>, Error>>()?;
             let outputs = dto.outputs;
 
             let mut builder = Self::builder(network_id)
                 .with_creation_slot(dto.creation_slot)
                 .with_context_inputs(dto.context_inputs)
                 .with_inputs(dto.inputs)
-                .with_mana_allotments(mana_allotments)
+                .with_mana_allotments(dto.allotments)
```

Comment:
not related but only a small change ... can you delete line 596 and just do `dto.outputs` in line 604?

Reply:
Done